R/check_package.R

```diff
@@ -1,4 +1,22 @@
-#' A standardised test for R packages
+#' Run the complete set of standardised tests on a package
+#'
+#' A convenience function that runs all packages related tests in sequence.
+#' The details lists the relevant functions.
+#' When you fixed a problem, you can speed things up by running only the related
```

Comment:
I suppose this is what you meant?
```suggestion
#' After fixing a problem, you can quickly check if it is solved by running only the related
```